at_cli/lib/src/at_cli.dart

```diff
@@ -113,21 +161,23 @@ class AtCli {
     }
   }
 
-  String _handleError(AtLookUpException exception) {
-    return '${exception.errorCode}-${exception.errorMessage}';
-  }
-
   /// Method to execute command
   /// input - command and isAuth
   /// return value - command response
-  Future<String> executeCommand(String command, {bool isAuth = false}) async {
+  Future<String> executeCommand(
+      String currentAtSign, AtCliPreference atCliPreference, String command,
```

Comment:
check whether you can remove atCliPreference from execute command..init outside execute command